Desktop/Regression_Test_Plan_Minor_Release.md

```diff
@@ -209,14 +208,8 @@ TestID | Test Case | Steps to reprouce| Expected Result | Result             | R
 18 :robot: | Public link a file and download it | 1) right click a file - open the share menu; 2) Create and copy public link to clipboard; 3) paste link in a web browser; 4) download | The file can be downloaded | :heavy_check_mark: | tst_sharing        |
 19 :robot: | Public link a folder and download it | 1) right click a folder - open the share menu; 2) Create public link; 3) 3dots menu -> "copy public link to clipboard (direct download)"; 4) paste link in a web browser | A zip archive of the folder can be downloaded | :heavy_check_mark: | tst_sharing        |
 20 :robot: | Public link to a file and disable it after | use the sharing menu to delete the public link | The link reports File not found. | :heavy_check_mark: |  tst_sharing  |
-21 | Public link to a file and 'add to your owncloud' a user that belongs to the server | | The file is shared | :construction: |   |
-22 | ~Public link to a file and 'add to your owncloud' a user that belongs to the server and has been already shared another file~ | | The file is shared | :heavy_minus_sign: | That only tests the server                                                
-23 | Public link to a file and 'add to your owncloud' with a user that belongs to another server | | The file is shared | :construction: |   |
-24 | Public link to a file and 'add to your owncloud' with an LDAP user | | The file is shared |  :construction: |   |
-25 | ~Optionally Public link to a file and 'add to your owncloud' with an Active Directory user~ | | The file is shared |    :construction: |       |
-26 | ~Public link to a file and 'add to your owncloud' with a Shibboleth user autoprovisioned~ | | The file is shared | :heavy_minus_sign: | Shiboleth is not supported | 
-27 | Try to reshare a file originally received from an LDAP user | | The file can be reshared, if "Allow Share" was set, otherwise it cannot be reshared | :construction:  | | 
-28 | Public link to a file and 'add to your owncloud' with a user that belongs to another oC server below 8.2.X version | `docker run --rm -ti -p 8181:80 owncloud:8.1` | The file is added to the old server | :construction: |    |
+21 | Public link to a folder with 'Upload only (File Drop) | 1) classic web UI create the public link as another user,</br>2) as your user, paste the link to "add to your owncloud",</br>3) wait for sync, visit the folder on the deskop,</br>4) Rename a file in the folder | Rename fails, an error message is shown | :construction: | FIXME: 2.10.0 silently duplicates the file |
+27 | Try to reshare a file originally received from an LDAP user | | The file can be reshared, if "Allow Share" was set, otherwise it cannot be reshared | :construction:  | (2.9.0 issue https://github.com/owncloud/client/issues/8941 ) |
```

Comment:
```suggestion
22 | Try to reshare a file originally received from an LDAP user | | The file can be reshared, if "Allow Share" was set, otherwise it cannot be reshared | :construction:  | (2.9.0 issue https://github.com/owncloud/client/issues/8941 ) |
```